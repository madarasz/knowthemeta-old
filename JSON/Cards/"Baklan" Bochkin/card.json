{"title":"\"Baklan\" Bochkin","code":"26017","overTime":[{"cardtitle":"\"Baklan\" Bochkin","sidecode":"runner","faction":"criminal","cardpacktitle":"Downfall","dpnumber":1,"cyclenumber":26,"indecks":3,"intopdecks":1,"deckfraction":0.0833333358168602,"topdeckfraction":0.0833333358168602}],"top":[{"cardtitle":"Nero Severn: Information Broker","sidecode":"runner","faction":"criminal","cardpacktitle":"Democracy and Dogma","dpnumber":3,"cyclenumber":10,"indecks":1,"intopdecks":-1,"deckfraction":1.0,"topdeckfraction":-1.0},{"cardtitle":"419: Amoral Scammer","sidecode":"runner","faction":"criminal","cardpacktitle":"The Devil and the Dragon","dpnumber":4,"cyclenumber":21,"indecks":1,"intopdecks":-1,"deckfraction":0.1111111119389534,"topdeckfraction":-1.0},{"cardtitle":"Az McCaffrey: Mechanical Prodigy","sidecode":"runner","faction":"criminal","cardpacktitle":"Downfall","dpnumber":1,"cyclenumber":26,"indecks":1,"intopdecks":-1,"deckfraction":0.5,"topdeckfraction":-1.0}],"combos":[{"title":"Always Have a Backup Plan","pack":"Downfall","faction":"criminal","score":44},{"title":"Eater","pack":"Order and Chaos","faction":"anarch","score":33},{"title":"Passport","pack":"Honor and Profit","faction":"criminal","score":33},{"title":"Abagnale","pack":"System Core 2019","faction":"criminal","score":33},{"title":"Early Bird","pack":"Honor and Profit","faction":"criminal","score":33},{"title":"Sneakdoor Beta","pack":"System Core 2019","faction":"criminal","score":33},{"title":"Bank Job","pack":"System Core 2019","faction":"criminal","score":33},{"title":"Embezzle","pack":"Whispers in Nalubaale","faction":"criminal","score":26},{"title":"Special Order","pack":"System Core 2019","faction":"criminal","score":21},{"title":"Bukhgalter","pack":"Downfall","faction":"criminal","score":20}],"decks":[{"count":3,"dpnumber":1,"cyclenumber":26,"dptitle":"Downfall","deckLinks":["<a href=\"https://netrunnerdb.com/en/decklist/54953\" rel=\"nofollow\" target=\"_blank\">Tag In Tag Out 419 - Valencia Store Winner</a> by <em>pviz</em> (#1 / 8 at Store Championship at Cartoon Corp (Valencia))","<a href=\"https://netrunnerdb.com/en/decklist/54904\" rel=\"nofollow\" target=\"_blank\">A poem for a bad deck</a> by <em>Cliquil</em> (#20 / 23 at Nisei Store Championships, Bristol, Uk)","<a href=\"https://netrunnerdb.com/en/decklist/54957\" rel=\"nofollow\" target=\"_blank\">the good stuff</a> by <em>NoahTheDuke</em> (#12 / 12 at Louisville Game Shop Store Championship)"]}]}