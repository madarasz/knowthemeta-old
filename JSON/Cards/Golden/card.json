{"title":"Golden","code":"11025","overTime":[{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Blood Money","dpnumber":2,"cyclenumber":11,"indecks":1,"intopdecks":0,"deckfraction":0.02083333395421505,"topdeckfraction":0.0},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Escalation","dpnumber":3,"cyclenumber":11,"indecks":0,"intopdecks":0,"deckfraction":0.0,"topdeckfraction":0.0},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Intervention","dpnumber":4,"cyclenumber":11,"indecks":5,"intopdecks":1,"deckfraction":0.03246753290295601,"topdeckfraction":0.013698630034923553},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Martial Law","dpnumber":5,"cyclenumber":11,"indecks":3,"intopdecks":1,"deckfraction":0.017751479521393776,"topdeckfraction":0.012345679104328156},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Quorum","dpnumber":6,"cyclenumber":11,"indecks":7,"intopdecks":1,"deckfraction":0.020527860149741173,"topdeckfraction":0.005747126415371895},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Daedalus Complex","dpnumber":1,"cyclenumber":12,"indecks":0,"intopdecks":0,"deckfraction":0.0,"topdeckfraction":0.0},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Station One","dpnumber":2,"cyclenumber":12,"indecks":1,"intopdecks":0,"deckfraction":0.018518518656492233,"topdeckfraction":0.0},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Terminal Directive","dpnumber":1,"cyclenumber":13,"indecks":0,"intopdecks":0,"deckfraction":0.0,"topdeckfraction":0.0},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Earth's Scion","dpnumber":3,"cyclenumber":14,"indecks":2,"intopdecks":1,"deckfraction":0.015037594363093376,"topdeckfraction":0.015384615398943424},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Blood and Water","dpnumber":4,"cyclenumber":14,"indecks":0,"intopdecks":0,"deckfraction":0.0,"topdeckfraction":0.0},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Free Mars","dpnumber":5,"cyclenumber":14,"indecks":1,"intopdecks":0,"deckfraction":0.006329114083200693,"topdeckfraction":0.0},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Crimson Dust","dpnumber":6,"cyclenumber":14,"indecks":3,"intopdecks":1,"deckfraction":0.04054053872823715,"topdeckfraction":0.02857142873108387},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Revised Core Set","dpnumber":1,"cyclenumber":20,"indecks":2,"intopdecks":0,"deckfraction":0.006968641187995672,"topdeckfraction":0.0},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Sovereign Sight","dpnumber":1,"cyclenumber":21,"indecks":1,"intopdecks":0,"deckfraction":0.011363636702299118,"topdeckfraction":0.0},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Down the White Nile","dpnumber":2,"cyclenumber":21,"indecks":0,"intopdecks":0,"deckfraction":0.0,"topdeckfraction":0.0},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Council of the Crest","dpnumber":3,"cyclenumber":21,"indecks":1,"intopdecks":1,"deckfraction":0.006410256493836641,"topdeckfraction":0.013333333656191826},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"The Devil and the Dragon","dpnumber":4,"cyclenumber":21,"indecks":0,"intopdecks":0,"deckfraction":0.0,"topdeckfraction":0.0},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Whispers in Nalubaale","dpnumber":5,"cyclenumber":21,"indecks":0,"intopdecks":0,"deckfraction":0.0,"topdeckfraction":0.0},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Kampala Ascendent","dpnumber":6,"cyclenumber":21,"indecks":0,"intopdecks":0,"deckfraction":0.0,"topdeckfraction":0.0},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Reign and Reverie","dpnumber":1,"cyclenumber":22,"indecks":6,"intopdecks":1,"deckfraction":0.021052632480859756,"topdeckfraction":0.007092198356986046},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"R&R MWL 2.2","dpnumber":2,"cyclenumber":22,"indecks":1,"intopdecks":1,"deckfraction":0.004950494971126318,"topdeckfraction":0.009433962404727936},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Magnum Opus","dpnumber":1,"cyclenumber":23,"indecks":2,"intopdecks":1,"deckfraction":0.01600000075995922,"topdeckfraction":0.022727273404598236},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"System Core 2019","dpnumber":1,"cyclenumber":25,"indecks":10,"intopdecks":2,"deckfraction":0.03759398311376572,"topdeckfraction":0.016393441706895828},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Downfall","dpnumber":1,"cyclenumber":26,"indecks":1,"intopdecks":1,"deckfraction":0.00374531839042902,"topdeckfraction":0.008403361774981022},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Downfall MWL 3.2","dpnumber":2,"cyclenumber":26,"indecks":0,"intopdecks":0,"deckfraction":0.0,"topdeckfraction":0.0},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Downfall MWL 3.3","dpnumber":3,"cyclenumber":26,"indecks":9,"intopdecks":2,"deckfraction":0.03879310190677643,"topdeckfraction":0.019417475908994675},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Uprising Booster Pack","dpnumber":4,"cyclenumber":26,"indecks":2,"intopdecks":0,"deckfraction":0.005434782709926367,"topdeckfraction":0.0},{"cardtitle":"Golden","sidecode":"runner","faction":"criminal","cardpacktitle":"Uprising","dpnumber":5,"cyclenumber":26,"indecks":0,"intopdecks":0,"deckfraction":0.0,"topdeckfraction":0.0}],"top":[{"cardtitle":"Adam: Compulsive Hacker","sidecode":"runner","faction":"adam","cardpacktitle":"Data and Destiny","dpnumber":1,"cyclenumber":9,"indecks":8,"intopdecks":-1,"deckfraction":0.19512194395065308,"topdeckfraction":-1.0},{"cardtitle":"Khan: Savvy Skiptracer","sidecode":"runner","faction":"criminal","cardpacktitle":"Blood Money","dpnumber":2,"cyclenumber":11,"indecks":2,"intopdecks":-1,"deckfraction":1.0,"topdeckfraction":-1.0},{"cardtitle":"Kabonesa Wu: Netspace Thrillseeker","sidecode":"runner","faction":"shaper","cardpacktitle":"Down the White Nile","dpnumber":2,"cyclenumber":21,"indecks":1,"intopdecks":-1,"deckfraction":0.021276595070958138,"topdeckfraction":-1.0}],"combos":[{"title":"Peregrine","pack":"Escalation","faction":"criminal","score":64},{"title":"Saker","pack":"Intervention","faction":"criminal","score":26},{"title":"Logic Bomb","pack":"Whispers in Nalubaale","faction":"adam","score":23},{"title":"Whistleblower","pack":"Downfall","faction":"neutral","score":22},{"title":"Emergent Creativity","pack":"Down the White Nile","faction":"adam","score":16},{"title":"Keros Mcintyre","pack":"Blood and Water","faction":"criminal","score":14},{"title":"Jackpot!","pack":"Whispers in Nalubaale","faction":"neutral","score":14},{"title":"Brain Chip","pack":"Data and Destiny","faction":"adam","score":14},{"title":"Multithreader","pack":"Data and Destiny","faction":"adam","score":12},{"title":"RNG Key","pack":"Down the White Nile","faction":"neutral","score":11}],"decks":[{"count":0,"dpnumber":5,"cyclenumber":26,"dptitle":"Uprising","deckLinks":[]},{"count":2,"dpnumber":4,"cyclenumber":26,"dptitle":"Uprising Booster Pack","deckLinks":["<a href=\"https://netrunnerdb.com/en/decklist/57203\" rel=\"nofollow\" target=\"_blank\">Flock of Seagulls 3 (MWL 3.4)</a> by <em>Atomikrin</em> (#7 / 7 at Concord Monthly GNK)","<a href=\"https://netrunnerdb.com/en/decklist/57203\" rel=\"nofollow\" target=\"_blank\">Flock of Seagulls 3 (MWL 3.4)</a> by <em>Atomikrin</em> (#11 / 11 at Gamescape October GNK)"]},{"count":9,"dpnumber":3,"cyclenumber":26,"dptitle":"Downfall MWL 3.3","deckLinks":["<a href=\"https://netrunnerdb.com/en/decklist/56556\" rel=\"nofollow\" target=\"_blank\">Lolipop Adam - 2nd Sheffield Regional</a> by <em>Blonde Haired Hacker Girl</em> (#2 / 14 at Sheffield Regionals)","<a href=\"https://netrunnerdb.com/en/decklist/56348\" rel=\"nofollow\" target=\"_blank\">Lolipop Adam - 1st York July GNK 5 player Round Robin</a> by <em>Blonde Haired Hacker Girl</em> (#1 / 5 at Critical Run July)","<a href=\"https://netrunnerdb.com/en/decklist/56556\" rel=\"nofollow\" target=\"_blank\">Lolipop Adam - 2nd Sheffield Regional</a> by <em>Blonde Haired Hacker Girl</em> (#6 / 20 at Derby Regional)","<a href=\"https://netrunnerdb.com/en/decklist/56728\" rel=\"nofollow\" target=\"_blank\">See Wu, Net Cowboy (again)</a> by <em>Dirjel</em> (#4 / 10 at Phoenix Regionals)","<a href=\"https://netrunnerdb.com/en/decklist/56925\" rel=\"nofollow\" target=\"_blank\">Loliclip Adam - 4th Mallorca Regional</a> by <em>Corbec</em> (#4 / 10 at Mallorca NISEI Regional Championship at Parabellum)","<a href=\"https://netrunnerdb.com/en/decklist/56589\" rel=\"nofollow\" target=\"_blank\">Lolipop Adam - 1st York July GNK 5 player Round Robin</a> by <em>pord</em> (#8 / 20 at Derby Regional)","<a href=\"https://netrunnerdb.com/en/decklist/56654\" rel=\"nofollow\" target=\"_blank\">York Regionals - Adam</a> by <em>Havvy</em> (#17 / 38 at York Regionals / Aki Birthday :cake:)","<a href=\"https://netrunnerdb.com/en/decklist/56540\" rel=\"nofollow\" target=\"_blank\">The Heartbroken Adam (1-3 SD Regional) </a> by <em>Insanit</em> (#19 / 34 at San Diego Regionals 2019 at At Ease Games)","<a href=\"https://netrunnerdb.com/en/decklist/56589\" rel=\"nofollow\" target=\"_blank\">Lolipop Adam - 1st York July GNK 5 player Round Robin</a> by <em>pord</em> (#10 / 14 at Sheffield Regionals)"]},{"count":0,"dpnumber":2,"cyclenumber":26,"dptitle":"Downfall MWL 3.2","deckLinks":[]},{"count":1,"dpnumber":1,"cyclenumber":26,"dptitle":"Downfall","deckLinks":["<a href=\"https://netrunnerdb.com/en/decklist/55482\" rel=\"nofollow\" target=\"_blank\">See Wu, Net Cowboy</a> by <em>Dirjel</em> (#1 / 5 at Phoenix Store Championship)"]},{"count":10,"dpnumber":1,"cyclenumber":25,"dptitle":"System Core 2019","deckLinks":["<a href=\"https://netrunnerdb.com/en/decklist/54725\" rel=\"nofollow\" target=\"_blank\">Tempo 419</a> by <em>pord</em> (#1 / 8 at Network Crash6)","<a href=\"https://netrunnerdb.com/en/decklist/54109\" rel=\"nofollow\" target=\"_blank\">Fall Guy Is a Crutch</a> by <em>elidelajandro</em> (#1 / 6 at Black Diamond Games: Nisei Standard)","<a href=\"https://netrunnerdb.com/en/decklist/54216\" rel=\"nofollow\" target=\"_blank\">The Resurgent Adam</a> by <em>Insanit</em> (#5 / 11 at GK Santa Clara Store Championship)","<a href=\"https://netrunnerdb.com/en/decklist/54216\" rel=\"nofollow\" target=\"_blank\">The Resurgent Adam</a> by <em>Insanit</em> (#6 / 11 at Reboot: Gamescape Netrunner Returns)","<a href=\"https://netrunnerdb.com/en/decklist/54502\" rel=\"nofollow\" target=\"_blank\">Adam 3.0 v3</a> by <em>pord</em> (#8 / 14 at Derby Store Championship)","<a href=\"https://netrunnerdb.com/en/decklist/54769\" rel=\"nofollow\" target=\"_blank\">Blueberry Diesel When?</a> by <em>HiddenAway</em> (#8 / 13 at Liverpool Store Championship 2019)","<a href=\"https://netrunnerdb.com/en/decklist/54725\" rel=\"nofollow\" target=\"_blank\">Tempo 419</a> by <em>pord</em> (#4 / 5 at Network Crash7)","<a href=\"https://netrunnerdb.com/en/decklist/54655\" rel=\"nofollow\" target=\"_blank\">Comet Ken v5</a> by <em>HiddenAway</em> (#21 / 26 at York Store Champisonship - Netrunner)","<a href=\"https://netrunnerdb.com/en/decklist/54236\" rel=\"nofollow\" target=\"_blank\">Ghost of Christmas Future</a> by <em>Tamijo</em> (#19 / 20 at Netrunner GNK - NISEI Q1 kit)","<a href=\"https://netrunnerdb.com/en/decklist/54417\" rel=\"nofollow\" target=\"_blank\">Security Geist</a> by <em>Menghini</em> (#8 / 8 at Netrunner Spring Fever - Store Championship)"]},{"count":2,"dpnumber":1,"cyclenumber":23,"dptitle":"Magnum Opus","deckLinks":["<a href=\"https://netrunnerdb.com/en/decklist/52961\" rel=\"nofollow\" target=\"_blank\">The Glorious Adam (bay area gnk winner) </a> by <em>Insanit</em> (#1 / 15 at Meat Damage, a farewell to netrunner)","<a href=\"https://netrunnerdb.com/en/decklist/52961\" rel=\"nofollow\" target=\"_blank\">The Glorious Adam (bay area gnk winner) </a> by <em>Insanit</em> (#2 / 6 at At Ease November GNK)"]},{"count":1,"dpnumber":2,"cyclenumber":22,"dptitle":"R&R MWL 2.2","deckLinks":["<a href=\"https://netrunnerdb.com/en/decklist/52554\" rel=\"nofollow\" target=\"_blank\">B O R E A N A Z - World's top Adam (28th place)</a> by <em>frankbsad</em> (#28 / 403 at Magnum Opus (Worlds 2018))"]},{"count":6,"dpnumber":1,"cyclenumber":22,"dptitle":"Reign and Reverie","deckLinks":["<a href=\"https://netrunnerdb.com/en/decklist/52117\" rel=\"nofollow\" target=\"_blank\">Engolokit</a> by <em>b3ar</em> (#2 / 7 at Netrunner LCG Turnier Abend)","<a href=\"https://netrunnerdb.com/en/decklist/52142\" rel=\"nofollow\" target=\"_blank\">Diversion of Bombs - UK Nats 2018 Best Adam (just)</a> by <em>pord</em> (#51 / 132 at UK Nationals 2018)","<a href=\"https://netrunnerdb.com/en/decklist/51644\" rel=\"nofollow\" target=\"_blank\">Run for the Money</a> by <em>D4KEN</em> (#7 / 16 at Netrunner Regional 2018)","<a href=\"https://netrunnerdb.com/en/decklist/52209\" rel=\"nofollow\" target=\"_blank\">UKs 2nd Best Adam</a> by <em>Hammers</em> (#58 / 132 at UK Nationals 2018)","<a href=\"https://netrunnerdb.com/en/decklist/52135\" rel=\"nofollow\" target=\"_blank\">UKs Worst Adam</a> by <em>Njabbskolt</em> (#60 / 132 at UK Nationals 2018)","<a href=\"https://netrunnerdb.com/en/decklist/52206\" rel=\"nofollow\" target=\"_blank\">Los Derez Efforts NUMBER 1 LOS@UK Nationals</a> by <em>ClenchMortified</em> (#116 / 132 at UK Nationals 2018)"]},{"count":0,"dpnumber":6,"cyclenumber":21,"dptitle":"Kampala Ascendent","deckLinks":[]},{"count":0,"dpnumber":5,"cyclenumber":21,"dptitle":"Whispers in Nalubaale","deckLinks":[]},{"count":0,"dpnumber":4,"cyclenumber":21,"dptitle":"The Devil and the Dragon","deckLinks":[]},{"count":1,"dpnumber":3,"cyclenumber":21,"dptitle":"Council of the Crest","deckLinks":["<a href=\"https://netrunnerdb.com/en/decklist/49736\" rel=\"nofollow\" target=\"_blank\">PoultryGeist: 4th NZ Nationals</a> by <em>EthosNZ</em> (#4 / 21 at New Zealand ANR Nationals Tournament)"]},{"count":0,"dpnumber":2,"cyclenumber":21,"dptitle":"Down the White Nile","deckLinks":[]},{"count":1,"dpnumber":1,"cyclenumber":21,"dptitle":"Sovereign Sight","deckLinks":["<a href=\"https://netrunnerdb.com/en/decklist/48082\" rel=\"nofollow\" target=\"_blank\">Steves Gauntlet (1-3)</a> by <em>Nurgleprobe</em> (#16 / 18 at Alphaspel Store Championship 2018)"]},{"count":2,"dpnumber":1,"cyclenumber":20,"dptitle":"Revised Core Set","deckLinks":["<a href=\"https://netrunnerdb.com/en/decklist/47399\" rel=\"nofollow\" target=\"_blank\">Auf Los geht's LOS!</a> by <em>dome_</em> (#6 / 10 at Achievement Tournament #1 - beginner friendly)","<a href=\"https://netrunnerdb.com/en/decklist/47212\" rel=\"nofollow\" target=\"_blank\">Party Birds in London</a> by <em>GarfieldMew</em> (#8 / 11 at The Hobbit Hole Store Championship)"]},{"count":3,"dpnumber":6,"cyclenumber":14,"dptitle":"Crimson Dust","deckLinks":["<a href=\"https://netrunnerdb.com/en/decklist/45164\" rel=\"nofollow\" target=\"_blank\">Prepaid Duck Hunt (2nd place GNK)</a> by <em>JDC_Wolfpack</em> (#2 / 7 at Atomic Empire September GNK)","<a href=\"https://netrunnerdb.com/en/decklist/46900\" rel=\"nofollow\" target=\"_blank\">Criminal: Khan (Gaming vs Cancer 2017)</a> by <em>protolich</em> (#17 / 34 at GvsC Android: Netrunner 2017)","<a href=\"https://netrunnerdb.com/en/decklist/45181\" rel=\"nofollow\" target=\"_blank\">Just Leela...or not? (PL version)</a> by <em>TwadaCZ</em> (#27 / 40 at Android: Netrunner OMP 2017)"]},{"count":1,"dpnumber":5,"cyclenumber":14,"dptitle":"Free Mars","deckLinks":["<a href=\"https://netrunnerdb.com/en/deck/view/972210\" rel=\"nofollow\" target=\"_blank\">Lost Vegas</a> by <em>binarymelon</em> (#7 / 7 at The Comic Hunter 2017 Q2 Tournament Kit)"]},{"count":0,"dpnumber":4,"cyclenumber":14,"dptitle":"Blood and Water","deckLinks":[]},{"count":2,"dpnumber":3,"cyclenumber":14,"dptitle":"Earth's Scion","deckLinks":["<a href=\"https://netrunnerdb.com/en/decklist/43533\" rel=\"nofollow\" target=\"_blank\">Leela Cortez (2nd DC Regionals)</a> by <em>netdeckdane</em> (#2 / 37 at DC Netrunner Regional Championship)","<a href=\"https://netrunnerdb.com/en/decklist/43832\" rel=\"nofollow\" target=\"_blank\">Just Leela...or not? (HU version)</a> by <em>TwadaCZ</em> (#12 / 21 at Hungarian Regionals 2017)"]},{"count":0,"dpnumber":1,"cyclenumber":13,"dptitle":"Terminal Directive","deckLinks":[]},{"count":1,"dpnumber":2,"cyclenumber":12,"dptitle":"Station One","deckLinks":["<a href=\"https://netrunnerdb.com/en/deck/view/917273\" rel=\"nofollow\" target=\"_blank\">APOCALOS</a> by <em>MrGlue</em> (#11 / 12 at 13th dae-gu store championship)"]},{"count":0,"dpnumber":1,"cyclenumber":12,"dptitle":"Daedalus Complex","deckLinks":[]},{"count":7,"dpnumber":6,"cyclenumber":11,"dptitle":"Quorum","deckLinks":["<a href=\"https://netrunnerdb.com/en/deck/view/848953\" rel=\"nofollow\" target=\"_blank\">geist</a> by <em>sirris</em> (#3 / 16 at 2017 Shuffle and Cut Store Championship)","<a href=\"https://netrunnerdb.com/en/decklist/41216\" rel=\"nofollow\" target=\"_blank\">Fisk & Teller 2.0</a> by <em>haxle</em> (#6 / 13 at Game knight store championship Winnipeg)","<a href=\"https://netrunnerdb.com/en/decklist/41217\" rel=\"nofollow\" target=\"_blank\">Birds of Catterjee Rock</a> by <em>zendealer</em> (#4 / 8 at Max Xp GNK)","<a href=\"https://netrunnerdb.com/en/decklist/38534\" rel=\"nofollow\" target=\"_blank\">Get Rich or Die Tryin'</a> by <em>Facecheck</em> (#12 / 20 at Leisure Games)","<a href=\"https://netrunnerdb.com/en/deck/view/862950\" rel=\"nofollow\" target=\"_blank\">Fast Leela GNK</a> by <em>D4KEN</em> (#4 / 6 at Game Night Kit, Spieleburg Göttingen)","<a href=\"https://netrunnerdb.com/en/decklist/40849\" rel=\"nofollow\" target=\"_blank\">Gabe v2</a> by <em>GarfieldMew</em> (#7 / 10 at Chatteris - The Hobbit Hole - SC)","<a href=\"https://netrunnerdb.com/en/deck/view/862750\" rel=\"nofollow\" target=\"_blank\">Waltzworm Leela</a> by <em>Cluster Fox</em> (#15 / 21 at Antwerp Store Championship)"]},{"count":3,"dpnumber":5,"cyclenumber":11,"dptitle":"Martial Law","deckLinks":["<a href=\"https://netrunnerdb.com/en/deck/view/839363\" rel=\"nofollow\" target=\"_blank\">Andy 2017</a> by <em>3Jane</em> (#1 / 15 at Ludus Belli GNK Diciembre)","<a href=\"https://netrunnerdb.com/en/deck/view/839272\" rel=\"nofollow\" target=\"_blank\">Reading SC / Andromeda</a> by <em>twisty_b</em> (#20 / 56 at Reading Store Championship)","<a href=\"https://netrunnerdb.com/en/decklist/39036\" rel=\"nofollow\" target=\"_blank\">Power Nexus Andy from @Zeromus</a> by <em>bubbathegoat</em> (#12 / 16 at Kult Spiele Store Championship)"]},{"count":5,"dpnumber":4,"cyclenumber":11,"dptitle":"Intervention","deckLinks":["<a href=\"https://netrunnerdb.com/en/decklist/39176\" rel=\"nofollow\" target=\"_blank\">Go Bird or Go Home</a> by <em>ocelotwildly</em> (#4 / 32 at Charity Gift II)","<a href=\"https://netrunnerdb.com/en/decklist/39147\" rel=\"nofollow\" target=\"_blank\">Peaking at the gifts</a> by <em>sythic</em> (#11 / 32 at Charity Gift II)","<a href=\"https://netrunnerdb.com/en/deck/view/815953\" rel=\"nofollow\" target=\"_blank\">SC Jönköping Runner 2</a> by <em>ingenfakir</em> (#7 / 13 at Store Championship Jönköping)","<a href=\"https://netrunnerdb.com/en/deck/view/825963\" rel=\"nofollow\" target=\"_blank\">Derez-Rerez Silhouette (Reading Jank GNK)</a> by <em>noahwp</em> (#20 / 27 at Reading Jank Tournament)","<a href=\"https://netrunnerdb.com/en/decklist/39036\" rel=\"nofollow\" target=\"_blank\">Power Nexus Andy from @Zeromus</a> by <em>bubbathegoat</em> (#11 / 14 at 2nd BankJob Tournament (Running in a bank branch))"]},{"count":0,"dpnumber":3,"cyclenumber":11,"dptitle":"Escalation","deckLinks":[]},{"count":1,"dpnumber":2,"cyclenumber":11,"dptitle":"Blood Money","deckLinks":["<a href=\"http://www.acoo.net/deck/19714\" rel=\"nofollow\" target=\"_blank\">Kate Tenma</a> by <em>Alexander Harvey</em> (#52 / 61 at Canada Nationals 2016)"]}]}